can/parser.cc

```diff
@@ -275,8 +284,8 @@ void CANParser::UpdateValid(uint64_t sec) {
     }
 
     const bool missing = state.last_seen_nanos == 0;
-    const bool timed_out = (sec - state.last_seen_nanos) > state.check_threshold;
-    if (state.check_threshold > 0 && (missing || timed_out)) {
+    const bool timed_out = (sec - state.last_seen_nanos) > state.alive_threshold;
```

Comment:
Not quite sure how to use avg_freq to do the checking here. @adeebshihadeh: any suggestions?